config/projects/chef.rb

```diff
@@ -65,6 +65,7 @@
 # Chef Release version pinning
 # override :chef, version: "12.3.0"
 # override :ohai, version: "8.3.0"
+override :libffi, version: "3.2.1" if ppc64? || ppc64le?
```

Comment:
Any reason we can't move all platforms to libffi 3.2.1? It would be better to take on the pain now and fix things on other platforms.

At the very least let's move things up above the release pinning. Perhaps on line 51 right below the `cacerts` override